torch/_inductor/codegen/triton.py

```diff
@@ -2002,7 +2002,7 @@ def codegen_nodes(self, nodes):
         if schedule_log.isEnabledFor(logging.DEBUG):
             schedule_log.debug("Schedule:\n %s", node_schedule)
 
-        return self.codegen_node_schedule(node_schedule, numel, rnumel)
+        return self.codegen_node_schedule(nodes, node_schedule, numel, rnumel)
```

Comment:
You shouldn't need to pass nodes here.  The `node_schedule` has all the nodes if you just do:
```
nodes = filter(lambda n: n not in (EnableReduction, DisableReduction), node_schedule)
```

(The EnableReduction/DisableReduction items in the `node_schedule` can be used to check how many loops there are.)